client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/hooks/use-credentials-form.ts

```diff
@@ -142,10 +122,11 @@ export const useCredentialsForm = ( onSubmit: ( siteInfo?: UrlData ) => void ) =
 		control,
 		handleSubmit,
 		submitHandler,
-		accessMethod: watch( 'migrationType' ),
+		accessMethod,
 		isBusy,
-		getContinueButtonText,
```

Comment:
I extract it from the hook because it is a UI responsibility, instead of a hook responsibility